inst/examples/03-formats.Rmd

```diff
@@ -218,9 +231,210 @@ favicon: "favicon.ico"
+
+The `index.Rmd` file is required, and is also your first book chapter. It will be the homepage when you render the book.
+
+A YAML header in `index.Rmd` for a `bs4_book` would look like this:
+
+```yaml
+---
+title: "A Minimal Book Example"
+author: "John Doe"
+date: "`r Sys.Date()`"
+site: bookdown::bookdown_site
+output: bookdown::bs4_book
+url: https://bookdown.org/johndoe/bookdown-demo
+cover-image: cover.png
+description: |
+  This is a minimal example of using the bookdown package to write a book.
+  The output format for this example is bookdown::bs4_book.
+---
+```
+
+#### Callout blocks
+
+The `bs4_book` theme also includes special callout blocks with predefined styles. Available blocks are: `.rmdnote`, `.rmdcaution`, `.rmdimportant`, `.rmdtip`, and `.rmdwarning`. 
+
+You need to use the following syntax to use them: 
+
+````markdown
+::: {.rmdnote}
+The `bs4_book` theme also includes an `.rmdnote` callout block like this one.
+```{r collapse=TRUE}
+head(beaver1, n = 5)
+```
+:::
+````
+
+You may use Markdown syntax and inline code inside a block. When knitted, this will render as:
+
+```{r fig.align='center', echo=FALSE, include=identical(knitr:::pandoc_to(), 'html'), fig.alt="A special rmd-note callout block."}
+knitr::include_graphics('images/rmd-note.png', dpi = NA)
+```
+
+For LaTeX output, only the content of these blocks will be shown with no special styling as for HTML. It is up to the user to define the appearance of these blocks for LaTeX output using custom environments. See this section in The R Markdown Cookbook: <https://bookdown.org/yihui/rmarkdown-cookbook/custom-blocks.html> for a how-to.
+
+
+#### HTML metadata
+
+HTML `<meta>` tags will be set based on Pandoc's variables set in `index.Rmd`:
+
+ * `pagetitle` (or `title` if unset) will be used as `og:title`
+ and `twitter:title` content.
+ * `description` will be used as `description`, `og:description` and
+ `twitter:description` content.
+ * `url` will be used as `og:url` and as base for `cover-image`
+ * `cover-image` is the path to your local cover image, and will be used as
+ `og:image`, appended to `url` value. `cover-image` can also take an absolute url.
+ * `twitter-handle` will be used as `twitter:site` content.
+
+If these values are set, then the associated `<meta>` tag will be set in the
+resulting HTML file. The provided values will be used asis in `index.html`.
+They will then be tweaked for each chapter to produce different metadata for h
+page as expected by social media:
+
+* All `description` related meta will be auto generated from page content,
+* `og:url` will be set to the specific HTML page url,
+* All the other `<meta>` will stay the same.
```

Comment:
I feel these two bullets are too much implementation detail. Do users really need to know this information?

Same thing for the next bullet list (the two meta tags that are not configurable).

Reply:
@yihui these were added to the documentation reference page, and I didn't want to delete that info because (as this is the main doc) it needs to be stated/saved somewhere. I'll move these to a footnote.